--- /usr/share/initramfs-tools/scripts/casper-bottom/30accessibility.old	2021-02-17 12:15:07.828966514 +0100
+++ usr/share/initramfs-tools/scripts/casper-bottom/30accessibility	2021-02-17 12:21:19.644455036 +0100
@@ -33,9 +33,11 @@
 			/root/usr/bin/casper-a11y-enable -initramfs -user="$USERNAME" -chroot=/root -dconf magnifier
                         ;;
 		# Blindness
-		access=v3)
+		access=blind)
 			NO_A11Y_DESKTOP_FILE=1
 			/root/usr/bin/casper-a11y-enable -initramfs -user="$USERNAME" -chroot=/root -dconf blindness
+			sed -i -e 's/idle-delay=uint32 300/idle-delay=uint32 0/' /root/usr/share/glib-2.0/schemas/90_belin.gschema.override;sed -i -e 's/idle-delay=5/idle-delay=120/' /root/usr/share/glib-2.0/schemas/90_belin.gschema.override;/root/usr/bin/glib-compile-schemas /root/usr/share/glib-2.0/schemas;/root/usr/bin/snap_torol.sh
+			echo "cat /etc/dconf/db/keybinding.d/00-keibinding|dconf load /" >>/root/home/$USERNAME/.profile
                         ;;
 		# Braille
 		braille=ask)
--- usr/lib/ubiquity/target-config/30accessibility.old	2021-02-17 12:37:47.039606859 +0100
+++ usr/lib/ubiquity/target-config/30accessibility	2021-02-17 14:51:34.435881259 +0100
@@ -46,9 +46,16 @@
 			exit
                         ;;
 		# Blindness
-		access=v3)
+		access=blind)
 			copy_orca_config
 			/usr/bin/casper-a11y-enable -login -user="$TARGET_USERNAME" -chroot=/target -dconf blindness
+			cp -r /home/$USERNAME/.local/share/orca /target/etc/skel/.local/share
+			cp -r /home/$USERNAME/.local/share/orca /target/home/$TARGET_USERNAME/.local/share
+			cp -r /home/$USERNAME/.local/share/orca /target/var/lib/lightdm/.local/share
+			chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.local
+			chroot /target chown -R lightdm.lightdm /var/lib/lightdm/.local
+			chmod 755 /target/home/$TARGET_USERNAME/.local/share/orca
+			python3 /usr/bin/ertesito&&rm /target/usr/bin/ertesito
 			exit
                         ;;
 		# Braille
